mkdocs/utils.py

```diff
@@ -35,6 +36,39 @@
     text_type = unicode         # noqa
 
 
+def yaml_load(source, loader=yaml.Loader):
+    """
+    Custom yaml loader.
+
+    Load all strings as unicode.
+    http://stackoverflow.com/a/2967461/3609487
+    """
+
+    class Loader(loader):
+
+        """Custom Loader."""
+
+        pass
```

Comment:
Apologies if I came off sounding like you didn't know what a `pass` was for.  I was more just trying to wrap my head around the response from my understanding albeit incorrect understanding.  I've just always used the `pass` explicitly.  It makes sense though thinking about it, as docstrings are parsed as part of the function/class and not just as a stray comment.